tests/failpoints/cases/test_snap.rs

```diff
@@ -240,9 +249,27 @@ fn test_destroy_peer_on_pending_snapshot() {
 
     fail::remove(before_handle_normal_3_fp);
 
+    assert_eq!(pending_remove_is_triggered.load(Ordering::Acquire), true);
```

Comment:
It may possible that the peer is destroyed before the gc snap tick is triggered, how about `sleep_ms` between `fail::remove(apply_snapshot_fp)` and `fail::remove(before_handle_normal_3_fp)`?

Reply:
So before snap_mgr_gc_tick_interval, we have sleep_ms(100),  is it not enough to run a gc snap (Right now the snap_mgr_gc_tick_interval is 50ms) ?

Added 50ms anyway in case.

Reply:
Snapshot gc is start only after `fail::remove(apply_snapshot_fp)`